CONTRIBUTING.md

```diff
@@ -0,0 +1,290 @@
+
+## Table of Contents
+
+- [Contributing to Kuma](#contributing-to-kuma)
+  - [Table of Contents](#table-of-contents)
+  - [Where to seek for help?](#where-to-seek-for-help)
+  - [Where to report bugs?](#where-to-report-bugs)
+  - [Where to submit feature requests?](#where-to-submit-feature-requests)
+  - [Contributing](#contributing)
+    - [Improving the documentation](#improving-the-documentation)
+    - [Submitting a patch](#submitting-a-patch)
+      - [Git branches](#git-branches)
+      - [Commit atomicity](#commit-atomicity)
+      - [Commit message format](#commit-message-format)
+        - [Type](#type)
+        - [Subject](#subject)
+        - [Body](#body)
+        - [Footer](#footer)
+        - [Examples](#examples)
+      - [Writing tests](#writing-tests)
+    - [Contributor T-shirt](#contributor-t-shirt)
+
+## Where to seek for help?
+
+[Slack](https://kuma-mesh.slack.com) is the main chat channel used by the
+community and the maintainers of this project. If you do not have an
+existing account, please follow this [link](https://chat.kuma.io) to sign
+up for free.
+
+**Please avoid opening GitHub issues for general questions or help**, as those
+should be reserved for actual bug reports. The Kuma community is welcoming and
+more than willing to assist you on Slack!
+
+[Back to TOC](#table-of-contents)
+
+## Where to report bugs?
+
+Feel free to [submit an issue](https://github.com/Kong/kuma/issues/new) on
+the GitHub repository, we would be grateful to hear about it! Please make sure
+to respect the GitHub issue template, and include:
+
+1. A summary of the issue
+2. A list of steps to reproduce the issue
+3. The version of Kuma you encountered the issue with
+4. Your Kuma configuration, or the parts that are relevant to your issue
+
+If you wish, you are more than welcome to propose a patch to fix the issue!
+See the [Submit a patch](#submitting-a-patch) section for more information
+on how to best do so.
+
+[Back to TOC](#table-of-contents)
+
+## Where to submit feature requests?
+
+You can [submit an issue](https://github.com/Kong/kuma/issues/new) for feature
+requests. Please add as much detail as you can when doing so.
+
+You are also welcome to propose patches adding new features. See the section
+on [Submitting a patch](#submitting-a-patch) for details.
+
+[Back to TOC](#table-of-contents)
+
+## Contributing
+
+We welcome contributions of all kinds, you do not need to code to be helpful!
+All of the following tasks are noble and worthy contributions that you can
+make without coding:
+
+- Reporting a bug (see the [report bugs](#where-to-report-bugs) section)
+- Helping other members of the community on Slack
+- Fixing a typo in the code
+- Fixing a typo in the documentation at https://kuma.io/docs (see
+  the [documentation contribution](#improving-the-documentation) section)
+- Providing your feedback on the proposed features and designs
+- Reviewing Pull Requests
```

Comment:
Add: Writing a blog post on how you're using Kuma

Reply:
added 👍